internal/params/config.go

```diff
@@ -756,6 +791,16 @@ func (c *ChainConfig) IsValidatorCodeFix(epoch *big.Int) bool {
 	return isForked(c.ValidatorCodeFixEpoch, epoch)
 }
 
+func (c *ChainConfig) IsHIP30(epoch *big.Int) bool {
+	return isForked(c.HIP30Epoch, epoch)
+}
+
+// Towards the end of this epoch, shards 2 and 3 will start sending
```

Comment:
update this since now, we are going to send the balance at start of HIP30 hardfork epoch - 1